packages/stats-py/aqsha_stats/analyses/uji_ancova.py

```
@@ -0,0 +1,99 @@
+    table,
+)
+from ..io import listwise_numeric, require_arg
+from ._glm import between_subjects_table, effect_p, fit_type3, q
+
+
+def run(df: pd.DataFrame, args: dict) -> dict:
+    dependent = str(require_arg(args, "dependent"))
+    factor = str(require_arg(args, "factor"))
+    covariates = [str(c) for c in require_arg(args, "covariates")]
+    sub, warnings = listwise_numeric(
+        df, [dependent, factor] + covariates, numeric=[dependent] + covariates
```

Ruff (0.15.20)

[warning] Consider `[dependent, factor, *covariates]` instead of concatenation

Replace with `[dependent, factor, *covariates]`

(RUF005)